src/VerifyPIVController.js

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint complexity:[2, 10], max-params: [2, 11] */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/FormType',
+  'views/shared/FooterWithBackLink'
+],
+function (Okta, FormController, FormType, FooterWithBackLink) {
+
+  var _ = Okta._,
+      $ = Okta.$;
+  var { Util } = Okta.internal.util;
+
+  return FormController.extend({
+    className: 'mfa-verify verify-piv',
+    Model: {
+
+      save: async function () {
+        this.trigger('request');
+        var self = this,
+            data = {
+              fromURI: this.settings.get('relayState')
+            },
+            pivButton = this.settings.get('piv');
+
+        try {
+          await this.getCert(pivButton.certAuthUrl);
+          const res = await this.postCert(pivButton.certAuthUrl, data);
+          Util.redirect(res.redirectUrl);
+        } catch (err) {
+          self.trigger('error', self, err);
+        }
+      },
+
+      getCert: function (certAuthUrl) {
+        return $.get({
+          url: certAuthUrl,
+          xhrFields: {
+            withCredentials: true
+          },
+          beforeSend: function () {},
```

Comment:
do we need this empty function? Are you overriding something?

Reply:
yea, im making this a simple request by not setting this header in the jquery-wrapper - https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/jquery-wrapper.js#L6